content/setup/create_servers.md

```diff
@@ -6,23 +6,25 @@ weight = 20
 toc = true
 +++
 
-### Using Terraform
+### Automated setup using Terraform
 Using Terraform (with the Packet provider), create two servers _tf-provisioner_ and _tf-worker_ attached to the same VLAN.
 
 - Clone the *tink* repository locally and switch to the *demo-v2* branch:
 ```sh
-$ git clone https://github.com/tinkerbell/tink.git
-$ cd tink
-$ git checkout demo-v2
-$ cd demo/terraform
+git clone https://github.com/tinkerbell/tink.git
+cd tink
+git checkout demo-v2
+cd demo/terraform
```

Comment:
Can we rename the directory to something else? 
`demo` does feel right here.

Reply:
You're right, I will first create a pr on tink to add the terraform files then I'll come back to this.